packages/view-types/layer-controller/src/styles.js

```diff
@@ -9,6 +9,16 @@ const useSpanStyles = makeStyles(() => ({
   },
 }));
 
+export const channelSliderStyles = makeStyles(theme => ({
```

Comment:
```suggestion
export const useChannelSliderStyles = makeStyles(theme => ({
```

Prefer to keep the `use` prefix naming convention for makeStyles